bundles/binding/org.openhab.binding.mios/examples/scripts/miosTransform.xslt

```diff
@@ -124,13 +125,16 @@ String   <xsl:value-of select="$DeviceNameFixed"/>DeviceStatus "<xsl:value-of se
 <xsl:when test="@service = 'urn:micasaverde-com:serviceId:AlarmPartition2'"      >AlarmPartition2</xsl:when>
 <xsl:when test="@service = 'urn:micasaverde-com:serviceId:Camera1'"              >Camera1</xsl:when>
 <xsl:when test="@service = 'urn:cd-jackson-com:serviceId:SystemMonitor'"         >SystemMonitor</xsl:when>
+<xsl:when test="@service = 'urn:cd-jackson-com:serviceId:Config'"                >CDJConfig</xsl:when>
```

Comment:
Exactly.  All MiOS Plugins create a UPnP service domain to work in., in order to store their state.

You'll see there categories, stand upnp-org ones, micasaverde-com ones and then custom ones by private developers.

When they're common in use, they get added to the item generator so it makes it easier for people to work with them (and have them auto migrate correctly instead of just 'String' maps... Which is the default